brage-import/src/test/java/no/sikt/nva/brage/migration/merger/PublicationInstanceMergerTest.java

```diff
@@ -0,0 +1,67 @@
+package no.sikt.nva.brage.migration.merger;
+
+import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValues;
+import static no.unit.nva.model.testing.PublicationGenerator.randomPublication;
+import static org.hamcrest.MatcherAssert.assertThat;
+import java.util.Set;
+import java.util.stream.Stream;
+import no.sikt.nva.brage.migration.model.PublicationRepresentation;
+import no.sikt.nva.brage.migration.record.Record;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.exceptions.InvalidIsbnException;
+import no.unit.nva.model.exceptions.InvalidIssnException;
+import no.unit.nva.model.exceptions.InvalidUnconfirmedSeriesException;
+import no.unit.nva.model.instancetypes.degree.DegreeBachelor;
+import no.unit.nva.model.instancetypes.degree.DegreeBase;
+import no.unit.nva.model.instancetypes.degree.DegreeLicentiate;
+import no.unit.nva.model.instancetypes.degree.DegreeMaster;
+import no.unit.nva.model.instancetypes.degree.DegreePhd;
+import no.unit.nva.model.instancetypes.degree.OtherStudentWork;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class PublicationInstanceMergerTest {
+
+    public static Stream<Arguments> emptyDegreeSupplier() {
```

Comment:
Tror du kan sende inn klassen direkte i stedet for å opprette en instans av den.

Reply:
I am sending not only class, but an object with null values.